scripts/convert_original_stable_diffusion_to_diffusers.py

```diff
@@ -143,8 +144,15 @@
         controlnet=args.controlnet,
     )
 
+    if args.half:
+        pipe.unet = pipe.unet.half()
```

Comment:
```suggestion
        pipe.to(dtype=torch.float16)
```

Reply:
ah! i didn't know that's possible! Thank you